content/en/docs/reference/glossary/duration.md

```diff
@@ -4,11 +4,18 @@ id: duration
 date: 2024-10-05
 full_link:
 short_description: >
-  A time interval specified as a string in the format accepted by Go's [time.Duration](https://pkg.go.dev/time), allowing for flexible time specifications using various units like seconds, minutes, and hours.
-aka:
+  A string value representing an amount of time.
 tags:
 - fundamental
 ---
-In Kubernetes APIs, a duration must be non-negative and is typically expressed with a suffix. 
-For example, `5s` for five seconds or `1m30s` for one minute and thirty seconds.
+A string value representing an amount of time.
 
+<!-- more -->
```

Comment:
This line could use a small correction, as the glossary entry isn't displayed correctly _(short and long descrption when expanded and collapsed)_. 

_Reference [preview page (here)](https://deploy-preview-48241--kubernetes-io-main-staging.netlify.app/docs/reference/glossary/?all=true&fundamental=true#term-duration)_.

```suggestion
<!--more-->
```

Reply:
Oops.

Maybe we should fix the glossary layout too!